test/parallel/test-buffer-bad-overload.js

```diff
@@ -8,7 +8,7 @@ assert.doesNotThrow(function() {
 
 assert.throws(function() {
   Buffer.from(10, 'hex');
-});
+}, /argument must not be a number/);
```

Comment:
@samshull  it helps tracking changes in the error message/type as those changes are breaking changes (semver-major).